README.md

```diff
@@ -2,6 +2,49 @@
 
 ## Dependencies and Recommendations
 
+### Makefile commands and usage
+
+This project uses a Makefile to automate common tasks such as setting up a virtual environment, installing dependencies, running linting with `pylint`, and cleaning up the environment.
+
+#### Available Commands
+
+1. **`make start`**:
+   - **Purpose**: Sets up the environment by creating a virtual environment (if it doesn't already exist), installs the dependencies, opens VS Code, and prints instructions for manually activating the virtual environment.
+   - **Steps**:
+     1. Creates a virtual environment called `venv` (if necessary).
+     2. Installs the project dependencies from `requirements.txt`.
+     3. Opens the project in VS Code.
+     4. Provides instructions to manually activate the virtual environment. (There is no workaround this, so the enviroment has to be manually activated after running `make start`).
+
+   - **Usage**:
+    `make start`
```

Comment:
It doesn't work for me. I've run `make clean` first and then `make start` but got this error:
![Screenshot 2024-09-12 at 13 19 48](https://github.com/user-attachments/assets/f16ec6c4-5697-4a51-baea-459543d93668)

Having a linter is useful, but this doesn't work for me. What was the system you tested on?

Tested on:
Macbook M3
OS: Sonoma 14.6.1

Reply:
Yes, forgot to add in the documentation that I havent tested it on unix based systems.

Reply:
I have only tested on windows so far.